### PR TITLE
Style all the things

### PR DESCRIPTION
**Pull request by mtedeschi**
Rolls in Jeff's work on revamping the sidebar as fixed position, and then adds in a bunch more styling. This includes the first pass at branding and styling from Warkulwiz.

### CODE REVIEW

File: python/cac_tripplanner/templates/base.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <![endif]-->
   </head>
 
-  <body class="homepage">
+  <body>
```
Comment (flibbertigibbet) [inline python/cac_tripplanner/templates/base.html]: If we remove the `homepage` class here, I don't think the contents of `_homepage.scss` will apply to anything.

File: src/app/scripts/cac/control/cac-control-itinerary-list.js [hunk 2]
```diff
@@ -70,13 +70,19 @@ CAC.Control.ItineraryList = (function ($, Handlebars, Utils) {
 
         var source = '{{#each itineraries}}' +
                 '<div class="block block-itinerary" data-itinerary="{{this.id}}">' +
+                '<div class="trip-numbers">'+
+                '<div class="trip-duration"> {{this.durationMinutes}} min</div>'  +
```
Comment (flibbertigibbet) [inline src/app/scripts/cac/control/cac-control-itinerary-list.js]: One space before `+`, here and line above.

File: src/app/scripts/cac/map/cac-map-templates.js [hunk 3]
```diff
@@ -62,7 +62,7 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
                 '<div class="modes"></div>',
                 '<h3>{{ d.name }}</h3>',
                 '<h5 class="distance-minutes"></h5>',
-                '<img src="{{#if d.wide_image}}{{ d.wide_image }}{{^}}https://placehold.it/300x150{{/if}}" width="300px" height="150px" />',
+                '<img src="{{#if d.wide_image}}{{ d.wide_image }}{{^}}http://placehold.it/300x150{{/if}}" />',
```
Comment (flibbertigibbet) [inline src/app/scripts/cac/map/cac-map-templates.js]: The https version of the image seems to be available; why switch it back?

Comment (kshepard) [inline src/app/scripts/cac/map/cac-map-templates.js]: Good call. We need all https from now on or else we'll get mixed-content warnings.

File: src/app/scripts/cac/control/cac-control-itinerary-list.js [hunk 4]
```diff
@@ -70,13 +70,19 @@ CAC.Control.ItineraryList = (function ($, Handlebars, Utils) {
...
+                '<div class="direction-icon">' +
                 ' {{modeIcon this}}' +
+                '</div>' +
                 '{{/each}}' +
-                '<span class="short-description"> via <b>{{this.via}}</b></span>' +
-                '<span class="trip-duration"> {{this.durationMinutes}} minutes,</span>'  +
-                '<span class="trip-distance"> {{this.distanceMiles}} mi.</span>' +
-                '<p><a class="itinerary" data-itinerary="{{this.id}}">View Directions</a></p>' +
+                '<span class="short-description"> via {{this.via}}</span>' +
+                '<a class="itinerary" data-itinerary="{{this.id}}">View Directions</a>' +
```
Comment (flibbertigibbet) [inline src/app/scripts/cac/control/cac-control-itinerary-list.js]: I'm going to add a space here between 'via' and the 'View Directions' link.
image

**Comment by flibbertigibbet** [comment]
+1, I'll make the minor changes noted above and merge it in.  Looks nice.

**Pull request merged**